packages/website/package.json

```diff
@@ -30,15 +30,16 @@
         "@emotion/react": "11.10.6",
         "@emotion/server": "11.10.0",
         "@faker-js/faker": "7.6.0",
-        "@mantine/carousel": "6.0.6",
-        "@mantine/core": "6.0.6",
-        "@mantine/dates": "6.0.6",
-        "@mantine/form": "6.0.6",
-        "@mantine/hooks": "6.0.6",
-        "@mantine/modals": "6.0.6",
-        "@mantine/next": "6.0.6",
-        "@mantine/notifications": "6.0.6",
-        "@mantine/prism": "6.0.6",
+        "@hookform/resolvers": "3.0.0",
+        "@mantine/carousel": "6.0.1",
+        "@mantine/core": "6.0.1",
+        "@mantine/dates": "6.0.1",
+        "@mantine/form": "6.0.1",
+        "@mantine/hooks": "6.0.1",
+        "@mantine/modals": "6.0.1",
+        "@mantine/next": "6.0.1",
+        "@mantine/notifications": "6.0.1",
+        "@mantine/prism": "6.0.1",
```

Comment:
Warning you have a regression here

Reply:
My mistake, I will correct that 😯. Thanks for pointing it out!